Directory.Packages.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <ItemGroup>
-    <PackageVersion Include="Autofac.Extensions.DependencyInjection" Version="4.2.2" />
+    <PackageVersion Include="Autofac.Extensions.DependencyInjection" Version="7.2.0" />
```

Comment:
This is the minimum version working with .NET 6 - no startup exceptions at all.

But should I update it to version 10 (also works without any code changes)?

Reply:
If you're confident the tests will find any breakages, upgrading it to the latest is fine.